### DISCUSSION TITLE
Automatic compilation of Gutenberg blocks

### DISCUSSION THREAD

**Discussion by igorlopasovsky** [tailpress/tailpress#46] [Ideas]
I am obviously struggling with JIT when using Gutenberg blocks. Wouldn't it be somehow possible to render all templates to temporary file first and then include this in the purge option in tailwind config? Safelist works fine for simple themes but not for complex designs with hundres of classes. I think this is the only real issue I've discovered so far.

Thank you for an amazing theme Jeffrey, really appreciate it.

**Comment by jeffreyvr** [comment]
I agree that purge as it is is not ideal. But because it is not a bug, I've transferred it to a discussion.

I've been thinking about a ways of trying enhance this, but so far no real solutions I'm afraid. The problem for example with figuring out the template files with `fast-glob` or something along those lines, and then passing those to purge list, makes it that it doesn't pick up on newly created files anymore.

**Reply by igorlopasovsky** [reply]
Thank you @jeffreyvr for reply.

What do you think about skipping the purge option completely? I suppose [it's not that bad](https://tailwindcss.com/docs/optimizing-for-production) _Using the default configuration, the development build of Tailwind CSS is 3566.2kB uncompressed, 289.2kB minified and compressed with Gzip, and 71.3kB when compressed with Brotli._
